aten/src/ATen/native/mps/kernels/UnaryKernel.metal

```diff
@@ -77,4 +74,3 @@ kernel void tanh_complex_kernel( device {0}2 *output [[buffer(0)]],
   auto tan_y = {0}(precise::tan(input[index].y));
   output[index] = complex_div({0}2(tanh_x, tan_y), {0}2({0}(1), tanh_x * tan_y));
```

Comment:
This needs to be replaced with an implicit template...

Reply:
How do I make sure this builds correctly? Do I need a full Xcode install?

Reply:
Figured out (with a full Xcode install -- now I see the compiler errors). Submitting an update version soon.